src/components/profile/style/profile.less

```diff
@@ -1,3 +1,6 @@
+/*library added until we integrate the icons*/
+@import url("https://netdna.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css");
```

Comment:
Please open an issue that we integrate the needed icon in our build and attribute the issue to @pfanguin or @pauloamado .